### DISCUSSION TITLE
Bug in header size when page width is full

### DISCUSSION THREAD

**Discussion by AndyRaphael** [imfing/hextra#308] [Q&A] [answered]
When changing the page width to full, the header remains at a smaller width.
This property applies when page size is full:
	.max-w-\[90rem\] {
	    max-width:90rem
	}

If I change it to max-width:100%, then header is correct. The property .max-w-full seems to be ignored, but seems to have been created for this page size.

**Comment by imfing** [comment, marked as answer]
https://imfing.github.io/hextra/docs/guide/configuration/#page-width

> Similarly, the width of the navbar and footer can be customized by the params.navbar.width and params.footer.width parameters.

**Reply by AndyRaphael** [reply]
Genius! Thank you
